src/components/settings/Account/Billing/Pricing.tsx

```diff
@@ -1,31 +1,29 @@
 import { SimpleGrid } from "@chakra-ui/react";
-import { AccountPlan } from "@3rdweb-sdk/react/hooks/useApi";
+import { Account, AccountPlan } from "@3rdweb-sdk/react/hooks/useApi";
 import { PricingCard } from "components/homepage/sections/PricingCard";
 import { useMemo } from "react";
 import { CONTACT_US_URL } from "utils/pricing";
 import { remainingDays } from "utils/date-utils";
 
 interface BillingPricingProps {
-  plan: AccountPlan;
+  account: Account;
   validPayment: boolean;
   paymentVerification: boolean;
   invalidPayment: boolean;
   loading: boolean;
-  canTrialGrowth?: boolean;
-  trialPeriodEndedAt?: string;
   onSelect: (plan: AccountPlan) => void;
 }
 
 export const BillingPricing: React.FC<BillingPricingProps> = ({
-  plan,
+  account,
   validPayment,
   paymentVerification,
   invalidPayment,
-  trialPeriodEndedAt,
   loading,
-  canTrialGrowth,
   onSelect,
 }) => {
+  const { plan, trialPeriodEndedAt } = account ?? {};
+  const canTrialGrowth = !trialPeriodEndedAt;
```

Comment:
looks cary because of the lots of conditionals put together

Reply:
... if all we want to know here in the end is `canTrialGrowth` can we just pass that in as a prop to begin with and do this logic above?

Reply:
Yup this makes sense, updated ✅